commit 284ecc7f3d8c86eabd06aad56f93fdfb727d3bbc
Author: Jie Yu <yujie.jay@gmail.com>
Date:   Mon Feb 3 13:38:47 2020 -0800

    dex: add Certificate used by Dex
    
    This is the Certificate object managed by cert-manager. It'll generate a
    certificate that will be used by Dex.

diff --git a/stable/dex-new/templates/certificate.yaml b/stable/dex-new/templates/certificate.yaml
new file mode 100644
index 0000000..247af5a
--- /dev/null
+++ b/stable/dex-new/templates/certificate.yaml
@@ -0,0 +1,24 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: dex
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    # We have to add the following annotation so that the upgrade does
+    # not fail with "already exists" error. See more details in:
+    # https://github.com/helm/helm/issues/5482
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  secretName: dex
+  issuerRef:
+    name: kubernetes-ca
+    kind: ClusterIssuer
+  commonName: dex
+  duration: 87600h
+  organization:
+    - D2iQ
+  dnsNames:
+    - dex-kubeaddons
+    - dex-kubeaddons.kubeaddons
+    - dex-kubeaddons.kubeaddons.svc.cluster.local
